config/env.sample

```diff
@@ -0,0 +1,5 @@
+PORT=5005
+CERT_FILE="config/public-cert.pem"
+KEY_FILE="config/private-key.pem"
```

Comment:
These two are already defaulted to the values you provide here in "proxy.js", so I see no particular need to have them here (unless we also remove the defaults and have it error when not provided via this default/sample config or otherwise…)